mhs/outbound/outbound/request/synchronous/handler.py

```diff
@@ -48,15 +145,41 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
             await self.invoke_default_workflow(from_asid, message_id, correlation_id, interaction_details, body, wf)
 
     def _parse_body(self):
+        try:
+            content_type = self.request.headers['Content-Type']
+        except KeyError as e:
+            logger.error('0011', 'Missing Content-Type header')
+            raise tornado.web.HTTPError(400, 'Missing Content-Type header', reason='Missing Content-Type header') from e
+        if content_type != 'application/json':
+            logger.error('0012', 'Unsupported content type in request. {ExpectedContentType} {ActualContentType}',
+                         {'ExpectedContentType': 'application/json', 'ActualContentType': content_type})
+            raise tornado.web.HTTPError(415,
+                                        'Unsupported content type. Only application/json request bodies are supported.',
+                                        reason='Unsupported content type. Only application/json request bodies are '
+                                               'supported.')
         body = self.request.body.decode()
         if not body:
             logger.error('0009', 'Body missing from request')
             raise tornado.web.HTTPError(400, 'Body missing from request', reason='Body missing from request')
-        return body
+        try:
+            # Parse the body as JSON and validate it against RequestBodySchema
+            parsed_body: request_body_schema.RequestBody = request_body_schema.RequestBodySchema().loads(body)
+        except json.JSONDecodeError as e:
+            logger.error('0013', 'Invalid JSON request body')
+            raise tornado.web.HTTPError(400, 'Invalid JSON request body', reason='Invalid JSON request body') from e
+        except marshmallow.ValidationError as e:
+            # e.messages is a nested dict of all the validation errors
+            validation_errors = str(e.messages)
```

Comment:
Note that I briefly looked into transforming `e.messages` into a string by iterating through it, but realised that it being nested (and the way it is nested) meant that this would be a bit complicated, and in any case, just running it through `str` gives us a formatted string of sorts.